worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -300,7 +300,9 @@ public void butcherBrush(Player player, LocalSession session,
                              @Switch(name = 'f', desc = "Also kill all friendly mobs (Applies the flags `-abgnpt`)")
                                  boolean killFriendly,
                              @Switch(name = 'r', desc = "Also destroy armor stands")
-                                 boolean killArmorStands) throws WorldEditException {
+                                 boolean killArmorStands,
+                             @Switch(name = 'w', desc = "Also destroy water mobs")
```

Comment:
IMO this should be `Also kill water mobs`, shouldn't it? "destroy" is just because armor stands are the only non-living creature.